app/views/admin/users/index.html.erb

```diff
@@ -1,37 +1,127 @@
-<% if FeatureFlag.enabled?(:member_index_view) %>
-  <%= render "admin/users/member_index" %>
```

Comment:
I deleted this partial now that we removed the Feature Flag and I've moved the contents of the partial into this template.